src/client/order-components/GoogleAPI.js

```diff
@@ -18,14 +23,39 @@ const formatAnswer = (mode, value) => {
   return winnerEstimate;
 };
 
-export const initMap = () => {
+// hacky-global scope variables to manage state for the requestRide
+const rideETA = {};
+const rideFare = {};
+
+export const requestRide = (mode) => {
+  let url = `http://localhost:8000/internal/requestRide`;
+  let body = {
+    ride: mode === 'Fare' ? rideFare : rideETA
+  };
+
+  fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        // token
+      },
+      body: JSON.stringify(body)
+    })
+    .then(res => res.json())
+    .then(data => {
+      console.log('success requestRide', data);
+      // render cancelRide and shareETA buttons for the user.
+    })
+    .catch(err => console.warn('error requestRide', err));
+}
+
+export const initMap = (cb) => {
   let origin_place_id = null;
   let destination_place_id = null;
   let travel_mode = 'DRIVING';
   let origin = {};
   let destination = {};
-  let userId = 1; // hardcoded.
-  //let carvisRideId;
+  let userId = 5;
```

Comment:
yeah already done in the current version - dropping DB between each test btw as our env variables were compromised in the travis logs